scripts/rebuildstaging

```diff
@@ -46,26 +41,18 @@ do
 done
 
 function rebuildstaging() {
-    echo "rebuilding staging branch, this might take a while..."
-    git-build-branch scripts/staging.yml "$@"
+    mkdir -p ./artifacts
```

Comment:
```suggestion
    mkdir ./artifacts
```
The parent directory is the current directory, `-p` is not needed.

Reply:
@skodde, `-p` will prevent it from failing if the directory already exists.